.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,48 @@
+name: "Triage Stale issues"
+
+on:
+  schedule:
+  - cron: "0 * * * *"
+
+jobs:
+  issue-cleanup:
+    runs-on: ubuntu-latest
+    name: Triage Stale Issues
+    steps:
+    - uses: aws-actions/stale-issue-cleanup@v4
+      with:
+        issue-types: issues
+        ancient-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer 
+          than five months. We encourage you to check if this is still an issue in the latest release. 
+          In the absence of more information, we will be closing this issue soon. 
+          If you find that this is still a problem, please feel free to provide a comment or upvote 
+          with a reaction on the initial post to prevent automatic closure. If the issue is already closed, 
+          please feel free to open a new one.
+        stale-issue-message: Hello 👋! It looks like this issue hasn’t been active in longer than two months. 
+          We encourage you to check if this is still an issue in the latest release. 
+          In the absence of more information, we will be closing this issue soon. 
+          If you find that this is still a problem, please feel free to provide a comment or upvote 
+          with a reaction on the initial post to prevent automatic closure. If the issue is already closed, 
+          please feel free to open a new one.
+        # These labels are required
+        stale-issue-label: closing-soon
+        exempt-issue-labels: bug, needs-triaging, should-be-fixed
+        response-requested-label: response-requested
+
+        # Don't set closed-for-staleness label to skip closing very old issues
+        # regardless of label
+        closed-for-staleness-label: closed-for-staleness
+
+        # Issue timing
+        days-before-stale: 60
+        days-before-close: 14
+        days-before-ancient: 150
+
+        # If you don't want to mark a issue as being ancient based on a
+        # threshold of "upvotes", you can set this here. An "upvote" is
+        # the total number of +1, heart, hooray, and rocket reactions
+        # on an issue.
+        minimum-upvotes-to-exempt: 2
```

Comment:
Will be interesting to see how this looks like in practice. I think the threshold may be a bit low - but we can bootstrap it with `2` for now, and then potentially increase to `5` or so.. 👍